Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Well of course it is based on the latest git master. Probably it will need a rebase now, but it had the latest changes. I have no idea, why this line is not in the master on other places, probably it got lost, when changing from jsp to xhtml.